src/Microsoft.Sbom.Api/Workflows/SBOMRedactionWorkflow.cs

```diff
@@ -50,44 +50,61 @@ public virtual async Task<bool> RunAsync()
         var sboms = await GetValidSbomsAsync();
```

Comment:
Should the call to `GetValidSbomsAsync` be inside the `try` block?

Reply:
Also appears to be out of date with the latest commits :)